test_scripts/API/PerformAudioPassThru/027_ATF_P_PerformAudioPassThru_UI_timeout_TTS_success.lua

```diff
@@ -0,0 +1,148 @@
+---------------------------------------------------------------------------------------------
```

Comment:
@maria-mih, please align the Test, TestStep_PerformAudioPassThru_UI_timeout_TTS_SUCESS